Zilliqa/zilliqa-developer
# US-336: Update documentations.

* Add scilla-parser, zilliqa-rs
* Add a pointer to where persistence is for testnet and mainnet
* Remove old information ("Upgrade to Zilliqa 8.3!")
* Sync APIs between ZQ1 LookupServer.cpp and the website (GetDSCommittee for example is not on the website)
* Add a "news" page.
* Add mentions of pyzil and go-zilliqa too!
* Add a pointer to [status.zilliqa.com](http://status.zilliqa.com/) on the front page so people can see what's up.
